wcfsetup/install/files/style/ui/dialog.scss

```diff
@@ -332,11 +332,10 @@ html[data-color-scheme="dark"] .dialog::backdrop {
 	color: var(--wcfContentText);
 	display: flex;
 	flex-direction: column;
-	max-height: var(--dialog-max-height);
-	max-width: var(--dialog-max-width);
+	max-height: calc(var(--dialog-max-height) - (2 * var(--dialog-padding)));
+	max-width: calc(var(--dialog-max-width) - (2 * var(--dialog-padding)));
```

Comment:
This doesn’t look right, the `box-sizing` should already account for the padding?